actions.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'app'
+
+# def create_person_option
+#   puts 'Do you want to create a Student (1) or a Teacher (2)? [Input the number]:'
+#   gets.chomp.to_i
+# end
+
+def create_student
+  puts "Enter student's age:"
+  age = gets.chomp.to_i
+  puts "Enter student's name:"
+  name = gets.chomp
+  puts 'Does the student have parent permission? (Y/N):'
+  parent_permission = gets.chomp.downcase == 'y'
+  create_student_message(Student.new(id: people.length + 1, age: age, name: name, parent_permission: parent_permission))
+end
+
+def create_teacher
+  puts "Enter teacher's age:"
+  age = gets.chomp.to_i
+  puts "Enter teacher's name:"
+  name = gets.chomp
+  puts "Enter teacher's specialization:"
+  specialization = gets.chomp
+  create_teacher_message(Teacher.new(age: age, name: name, specialization: specialization))
+end
```

Comment:
- [ Required ] I think it would be nice if you ensure that the logic for creating a `Student` and `Teacher` works as expected and without any error. Currently, when I try to create either a student or teacher I get the error below. Please see the screenshot below and rectify this. 
 I was neither able to create a `student` nor a `teacher`  so I could not test for `listing people`, `renting a book`, or `listing rentals`, please make sure these too work.

## Error in project
> Error when creating a teacher
![Screenshot from 2023-11-10 18-05-28](https://github.com/martinkarugaba/oop-school-library/assets/16392046/f498ebfe-88b5-4bb5-b168-bb8f53ea4ea4)

> Error when creating a student
![Screenshot from 2023-11-10 18-24-28](https://github.com/martinkarugaba/oop-school-library/assets/16392046/41f3fa53-c152-4ffc-b94e-f4f6c43e84c6)